Gopkg.toml

```diff
@@ -137,7 +137,7 @@ required = [
 
 [[constraint]]
   name = "github.com/dgraph-io/badger"
-  version = "=1.5.3"
+  version = "^1.6.0"
```

Comment:
Given breaking changes, are you sure we want ^ and not =?

Reply:
Good call, will fix